parquet-hadoop/pom.xml

```diff
@@ -79,6 +85,11 @@
       <artifactId>commons-pool</artifactId>
       <version>1.6</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.rdblue</groupId>
```

Comment:
com.github.rdblue seems like a temporary groupId to me, do you want to add this dependency as it is to parquet-format?

Reply:
This is how jitpack.io creates coordinates.